src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -337,8 +337,12 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Choose>
 
   <PropertyGroup Condition="'$(RoslynCompilerType)' == 'Core'">
+    <!-- In Full Framework MSBuild, RoslynTargetsPath originally points to the directory with Full Framework Roslyn assemblies. -->
+    <RoslynAssembliesPath Condition="'$(MSBuildRuntimeType)' != 'Core'">$(RoslynTargetsPath)</RoslynAssembliesPath>
```

Comment:
I think we had previously allowed user's to specify this path.  Should we honor their value if already set?